wireless_modules/libraries/abc.py

```diff
@@ -0,0 +1,2 @@
+def abstractmethod(f):
+    return f
```

Comment:
Also when using `from abc import abstractmethod`, would this user-created file conflict with the built-in Python module?

Reply:
Shouldn't conflict since there is no built-in module in Micropython for ABC. And the reason why I didn't comment the function was because I didn't write it (and I have heard you usually don't want to comment third-party code (since you want to leave it as it is so it can easily be compared and updated if needed). However, I maybe I'll link the source and perhaps describe why I'm using it at the start of the file.

Reply:
Yeah in general that's probably true but since this is unlikely to ever be updated anyway, and the file is very minimal and straightforward to understand as it is, the comments are probably worth it to avoid future confusion.

Reply:
ohhhhhhh, I actually never bothered to look at what a decorator does. I just knew I had to add it before certain functions in python. This makes total sense now.